jsconfig.json

```diff
@@ -0,0 +1 @@
+{"compilerOptions":{"target":"es6","experimentalDecorators":true},"exclude":["node_modules","bower_components","tmp","vendor",".git","dist"]}
```

Comment:
This looks like a file that is probably for your editor and thus should be removed from your PR and added to `.gitignore`.